blueye/sdk/logs.py

```diff
@@ -0,0 +1,150 @@
+from datetime import datetime
+
+import requests
+import tabulate
+
+
+class LogFile:
+    """
+    This class is a container for a log file stored on the drone
+
+    The drone lists the file name, max depth, start time, and file size for each log,
+    and you can show this information by printing the log object, eg. on a Pioneer
+    object called `p`:
+
+    ```
+    print(p.logs[0])
+    ```
+
+    or, if you want to display the header you can format the object with `withHeader`:
+
+    ```
+    print(f{p.logs[0]:withHeader})
+    ```
+
+    Calling the download() method on a log object will pull the CSV (Comma Separated
+    Value) file from the drone to your local filesystem.
+    """
+
+    def __init__(self, maxdepth, name, timestamp, binsize, ip):
+        self.maxdepth = maxdepth
+        self.name = name
+        self.timestamp = datetime.fromisoformat(timestamp)
+        self.binsize = binsize
+        self.downloadPath = "http://" + ip + "/logcsv/" + name
+        self._formattedValues = [
+            self.name,
+            self.timestamp.strftime("%d. %b %Y %H:%M"),
+            f"{self.maxdepth/1000:.2f} m",
+            self._humanReadableFilesize(),
+        ]
+
+    def _humanReadableFilesize(self):
+        suffix = "B"
+        num = self.binsize
+        for unit in ["", "Ki", "Mi"]:
+            if abs(num) < 1024.0:
+                return "%3.1f %s%s" % (num, unit, suffix)
```

Comment:
```suggestion
                return f"{num:3.1f} {unit}{suffix}"
```
f strings are a bit nicer to read